### ISSUE TITLE
Missing speech in victory road

### ISSUE THREAD

**Issue by Spazmoid1981** [MiSTer-devel/Arcade-IkariWarriors_MiSTer#4]
@RndMnkIII does my memory serve me correct that there was a laughing sample as the game starts with the spinning rock thing?

**Comment by RndMnkIII** [comment]
Exactly, there is no playback of ADPCM digital samples because the sound chip that plays them is the Yamaha MSX-AUDIO of which there is currently no core that implements this functionality.

**Comment by jfml2020** [comment]
I realized today while playing the MAME version that the floating head talks, but I switched to Mister FPGA and sure enough, it doesn't have a voice. Cheers.
